plugins/modules/dcnm_interface.py

```diff
@@ -76,6 +76,9 @@
       profile_pc:
         description:
           - Object profile which must be included for port channel interface configurations.
+          - NOTE: Though the key shown here is 'profile_pc' the actual key to be used in playbook 
```

Comment:
@mmudigon I had a chance to chat about this with Shangxin and we should instead update the description to include the type that this particular parameter is associated with.

The structure would look something like this:

```yaml
      profile:
        description:
          - Object profile used to include suboptions for configuring different interface types.
        suboptions:
          # ----------------- Common suboptions -----------------
          mode:
            description:
              - Interface mode
            choices: ['trunk', 'access', 'l3', 'monitor']
            type: str
            required: true
          # ----------------- Suboptions for type: pc -----------------
          members:
            description:
              - Member interfaces that are part of this port channel
              - Requires type parameter to be set to 'pc'
            type: list
            elements: str
            required: true
          # ----------------- Suboptions for type: vpc -----------------
          peer1_pcid:
            description:
              - Port channel identifier of first peer. If this object is not included, then the value defaults to the
                vPC identifier. This value cannot be changed once vPC is created
              - Requires type parameter to be set to 'vpc'
            type: int
            choices: [Min:1, Max:4096]
            default: Default value is the vPC port identifier